dashboard/models.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
I have not a strong opinion about this: I can move it if you feel strong about it. Let me know